app/models/menu_items_menu.rb

```diff
@@ -0,0 +1,20 @@
+class MenuItemsMenu < ActiveRecord::Base
```

Comment:
True @sampritipanda . As I mentioned in my PR description we should rethink the naming for models and tables associated with this workflow. I reused the same table name that was already in place.